content/blog/configuring-azure-ad-with-greenlake-cloud-platform-and-aruba-central.md

```diff
@@ -0,0 +1,236 @@
+---
+title: Configuring Azure AD as the SAML IDP with Greenlake Cloud Platform and Aruba Central
+date: 2022-05-09T12:04:30.750Z
```

Comment:
Suggest to set the date to today as it is a new blog post submitted to HPE Developer Community portal.